install_config/install/advanced_install.adoc

```diff
@@ -1644,9 +1655,27 @@ be deleted when the Elasticsearch pod terminates.
 There are three options for enabling cluster logging storage when using the
 advanced install:
 
+[discrete]
+[[advanced-installation-logging-storage-dynamic]]
+===== Option A: Dynamic
+
+Use the following variable if your {product-title} environment supports
+xref:../../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#install-config-persistent-storage-dynamically-provisioning-pvs[dynamic volume provisioning] for your cloud provider:
+
+----
+[OSEv3:vars]
+
+openshift_logging_storage_kind=dynamic
+----
+
 [discrete]
 [[advanced-installation-logging-storage-nfs-host-group]]
-===== Option A: NFS Host Group
+===== Option B: NFS Host Group
+
+[NOTE]
+====
+The use of NFS is experimental and not supported in {product-title}.
```

Comment:
@bmcelvee I would consider updating each of these boxes to be IMPORTANT boxes since it is calling out supportability.